social-ui/src/Meta.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from "react";
+
+//can't get the data to pass through I give up
```

Comment:
Your component does receive a prop 'article' If you inspect the component in chrome react dev tools you will be able to see more clearly how the props are organized, or console log this.props.